translate/v23tov30/v23tov30.go

```diff
@@ -518,6 +519,28 @@ func RemoveDuplicateFilesAndUnits(cfg old.Config) (old.Config, error) {
 		}
 	}
 
+	// Concat sshkey sections into the newest passwdUser in the list
+	// We make the assumption that there is only one user: core
+	// since that is the only supported user by design.
+	// It's technically possible, though, to have created another user
+	// during install time configs, since we only check the validity of
+	// the passwd section if it was changed. Explicitly error in that case.
+	if len(users) > 0 {
+		outUser := users[len(users)-1]
+		if outUser.Name != "core" {
+			return cfg, errors.Errorf("unexpected user with name: %v. Only the core user is supported.", outUser.Name)
```

Comment:
This is fine for the MCO but we shouldn't impose this restriction in ign-converter.

Reply:
:+1: I'll update this part so we can merge arbitrary users and when switching to use this func in the MCO, the check can still be performed there.